### ISSUE TITLE
Support for multiple collections

### ISSUE THREAD

**Issue by nobkd** [nuejs/nue#145]
Is it possible to define multiple collections?

E.g. I want to have a global collection for all pages (e.g. global navigation), and a sub-collection for only a specific subfolder (e.g. posts of a blog in a subfolder).

I know, that code-wise there can be always only one collection currently.
Are there plans to add support for multiple collections at the same time?

**Comment by tipiirai** [comment]
Indeed. This is not currently supported. I was thinking that having multiple collections on a single page is an edge case and rarely needed. Can you think of a good example site where we can see multiple (imaginary) collections in place? I'm thinking that global navigation is always a hand-picked list of key pages, but I'm probably wrong.

**Comment by nobkd** [comment]
Sorry for the late reply.

It's not really necessary, but I built my navigation in the following way, and if I want to use the files of only a sub-path, *I have to filter the global collection myself* (which also works :shrug:).

Here's my navigation example:

In the front-matter of my Markdown files, I have a key `cat` (category), which contains a key for every category and a value (int / number) for the position / order in the category.

Example front-matter:

```md
<!-- /index.md -->
---
title: Home Page
cat:
  header: 0
---

some interesting content
```
...

```md
<!-- /contact/index.md -->
---
title: Contact
cat:
  header: 5
---

text, contact form, etc.
```

Then I can filter the global collection by category name and order it by the value of that key.

Advantage: I don't have to care about the file path of markdown files. It does not matter, if they are a `index.md` (I shorten the path to end on the folders' `dir/` instead of `dir/index.html`) in a subfolder or not, regardless of where they are in the project. -> No manual path declarations, that might change over time

Disadvantage: The navigation is part of the Markdown files and is split up all over the project files -> data scattered everywhere

---

Then, I have more collections in sub-dirs, e.g. my blog example from before, where i want to list all the posts on the index page of the blog. As mentioned in the beginning, I could also filter out the relevant pages myself. but it obviously would be easier to just define a new collection for the current dir and subdirs, without losing the navigation from before on this page/subpages
